### ISSUE TITLE
Xenobio kill chamber is broken

### ISSUE THREAD

**Issue by Hexxproof** [tgstation/-tg-station#13313]
The vaccum tiles only work if the door never opens, and someone always fucking opens it, so it doesn't work.

**Comment by WJohn** [comment]
Simple exposure to space would probably be adequate. Maybe a couple of grilles to the left and right of the kill chamber? I dunno.

**Comment by Zilenan91** [comment]
What map?

**Comment by GunHog** [comment]
Add blast doors leading to space, instead of the window-grille. (I would assume he means Box, but he really should provide more information).

**Comment by KorPhaeron** [comment]
Boxstation

**Comment by Incoming5643** [comment]
It's a problem with any station where there's a door between the chamber and xenobio. This actually brings up a curious quirk of slimes.

They don't breathe, and they don't suffer due to pressure, it being cold is the ONLY thing that kills them in terms of atmosphere. So a true vacuum is harmless to them UNLESS the vacuum is also cold. Anyone who actually knows how space works can tell you that while space is very cold, it's also so sparsely filled with anything that absorb heat that anyone in it wouldn't freeze to death.

tl;dr: Space is only deadly to slimes because we use "hollywood space" rules instead of "real space" rules. The spoiled vacuum chamber shows how real space would effect slimes, in that it wouldn't.

**Comment by GunHog** [comment]
Instead of an vacuum chamber, you could either make it a chilling chamber, like the server room, or just blast doors that open to space.

**Comment by Bluespace** [comment]
Make it space tiles but put walkway grilles like the solars on there too. 
It looks pretty.

**Comment by Jalleo** [comment]
I prefergun hogs idea
